training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml

```diff
@@ -448,112 +448,112 @@ outputs:
 jobs:
   question_answering_model_import:
     type: command
-    component: azureml://registries/azureml/components/question_answering_model_import/versions/0.0.5
+    component: azureml://registries/azureml/components/question_answering_model_import/versions/0.0.6
     compute: ${{parent.inputs.compute_model_import}}
     inputs:
-      huggingface_id: ${{parent.inputs.huggingface_id}}
-      pytorch_model_path: ${{parent.inputs.pytorch_model_path}}
-      mlflow_model_path: ${{parent.inputs.mlflow_model_path}}
+      huggingface_id: '${{parent.inputs.huggingface_id}}'
```

Comment:
Are quotes required here or including them in the command will suffice?

Reply:
Probably putting them in command line will suffice, but few parameters like compute might still need it the compute name is allowed to have spaces in it.

Reply:
ok got it